src/components/Tools/Card.vue

```diff
@@ -0,0 +1,132 @@
+<template>
+  <div class="card" :style="cssVars" :class="{ 'card--collapsed': isCollapsed }">
+    <div class="card-top">
+      <button class="card-minspacer">
+        <img v-if="!isCollapsed" src="@/assets/images/minimize.svg" alt="minimize card" />
+        <img v-else src="@/assets/images/expand.svg" alt="expand card" />
+      </button>
+      <div class="card-name">{{ name }}</div>
+      <button class="card-minimize" @click="collapse">
+        <img v-if="!isCollapsed" src="@/assets/images/minimize.svg" alt="minimize card" />
+        <img v-else src="@/assets/images/expand.svg" alt="expand card" />
+      </button>
+    </div>
+    <div v-if="isCollapsed"></div>
+    <hr class="card-divider" v-if="!isCollapsed" />
+    <div class="card-content" v-if="!isCollapsed">
+      <slot></slot>
+    </div>
+  </div>
+</template>
+
+<script>
+import { defineComponent } from 'vue';
+
+export default defineComponent({
+  props: {
+    id: { type: String, required: true },
+    name: { type: String, required: true },
+    desiredWidth: { type: Number, required: false, default: 574 },
+    desiredHeight: { type: Number, required: false, default: 275 },
+  },
+
+  data() {
+    return {
+      isCollapsed: false,
+    };
+  },
+  mounted() {
+    const savedMinimize = localStorage.getItem(JSON.stringify(this.name));
+    this.isCollapsed = savedMinimize ? JSON.parse(savedMinimize) : false;
+  },
+  methods: {
+    collapse() {
+      this.isCollapsed = !this.isCollapsed;
+      localStorage.setItem(JSON.stringify(this.name), JSON.stringify(this.isCollapsed));
+    },
+  },
+  computed: {
+    cssVars() {
+      return {
+        '--w': `${this.desiredWidth}px`,
+        '--h': `${this.desiredHeight}px`,
+      };
+    },
+  },
+});
+</script>
+
+<style scoped lang="scss">
+@import '@/assets/scss/_variables.scss';
+
+.card {
+  width: var(--w);
+  height: var(--h);
+  box-sizing: border-box;
+  box-shadow: 0 0 10px 4px $boxShadowGray;
+  position: relative;
+  border-radius: 10px;
+  border-width: 0;
```

Comment:
Does anyone know why I need to set this to 0 explicitly? This is the highest scope (.card), so I don't know why it has a nonzero value by default.